store/account/account.go

```diff
@@ -60,32 +57,64 @@ func NewWithDB(db *gorm.DB) Store {
 	}
 }
 
-func (t *store) InsertERC20(code *model.ERC20) error {
+func (t *store) InsertERC20(code *model.ERC20, onlySubscribe bool) error {
```

Comment:
Do we need to build index for these erc20 related tables?

Reply:
Please ignore this comment. I just find the index tags declared in struct.

Reply:
OK